### DISCUSSION TITLE
Pressing down using a tablet (not using osu!lazer native tablet system) causes cursor to freeze or stutter

### DISCUSSION THREAD

**Discussion by YourLocalTI84** [ppy/osu#26875] [Q&A]
### Type

Game behaviour

### Bug description

When pressing down on a tablet with the pen, the cursor freezes or stutters for an instant. If held down, same occurs

### Screenshots or videos

https://twitter.com/i/status/1752474611197845813

### Version

2024.130.2-lazer

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14105480/compressed-logs.zip)

**Comment by bdach** [comment]
Did you disable mouse buttons in settings?

**Reply by YourLocalTI84** [reply]
Tried with and without disabling. Here is all my input settings
![image](https://github.com/ppy/osu/assets/51512544/5d09b8e4-3a55-4014-9399-7e02cc7498cc)
![image](https://github.com/ppy/osu/assets/51512544/8436ffb1-841c-4b1d-89e4-fc2b46754607)

**Comment by peppy** [comment]
Does turning off "high precision input mouse" with this?

![2024-02-19 11 12 26@2x](https://github.com/ppy/osu/assets/191335/d942d4b3-1c36-4cb6-b39d-03ab0f2f5e9e)

**Comment by Susko3** [comment]
From your video, it seems that pressing the tablet pen unconditionally "pauses" cursor movement. Could you record the same video again, but open global statistics (Ctrl+F2) and scroll down to the `Input - ` entries. You can upload the video here on github by just dragging it into the comment box.
